<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgDesaturate</class>
 <widget class="QWidget" name="WdgDesaturate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>194</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Desaturation method:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioLightness">
        <property name="text">
         <string>&amp;Lightness</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioLuminosityBT709">
        <property name="text">
         <string>Luminosity (ITU-R BT.&amp;709)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioLuminosityBT601">
        <property name="text">
         <string>Luminosity (ITU-R BT.&amp;601)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioAverage">
        <property name="text">
         <string>&amp;Average</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioMin">
        <property name="text">
         <string>&amp;Min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioMax">
        <property name="text">
         <string>M&amp;ax</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
